### DISCUSSION TITLE
Unable to scroll documentation window

### DISCUSSION THREAD

**Discussion by gennaro-tedesco** [folke/lua-dev.nvim#113] [Q&A] [answered]
I am using `neodev.nvim` in conjunction with `noice`, where in particular the lsp setup is configured as
```lua
require("noice").setup({
...
	lsp = {
		override = {
			["vim.lsp.util.convert_input_to_markdown_lines"] = true,
			["vim.lsp.util.stylize_markdown"] = true,
			["cmp.entry.get_documentation"] = true,
		},
```
and the scrolling keymaps are
```lua
vim.keymap.set("n", "<c-j>", function()
	if not require("noice.lsp").scroll(4) then
		return "<c-j>"
	end
end, { silent = true, expr = true })

vim.keymap.set("n", "<c-k>", function()
	if not require("noice.lsp").scroll(-4) then
		return "<c-k>"
	end
end, { silent = true, expr = true })
```
The `neodev` documentation window does not scroll (whereas all other completion sources do, for docs and signatures). Do I have to explicitly set up the scrolling for `neodev` as well or is it supposed to work out of the box (as probably taken over by `noice`) and I must have a mistake in my configuration, somewhere?

**Comment by folke** [comment, marked as answer]
Your keymaps are only for normal mode, are you trying to scroll in insert mode?

(I need to fix those mappings examples)

You probably want to change the `"n"` to `{"n", "i", "s"}`

**Reply by gennaro-tedesco** [reply]
Embarrassing from my side, of course. 

P. S. All other completion sources seem to still scroll fine even without specifying the insert modes: is it because they're added as completion sources and perhaps `nvim-cmp` handles this under the hood while selecting the entry you're after?

**Reply by folke** [reply]
yep, exactly. that's cmp.
I've just updated the Noice docs to include the other modes. That was an oversight on my side :)
